Mathlib/Data/Multiset/DershowitzManna.lean

```diff
@@ -10,9 +10,12 @@ import Mathlib.Logic.Relation
 /-!
 # Dershowitz-Manna ordering
 
-In this file we define the _Dershowitz-Manna ordering_ on multisets.
-We prove that, given a well-founded partial order on the underlying set,
-the Dershowitz-Manna ordering defined over multisets is also well-founded.
+In this file we define the _Dershowitz-Manna ordering_ on multisets. Specifically, for two multisets
+M and N over an underlying set S, M is smaller than N in the Dershowitz-Manna ordering if M can be
```

Comment:
```suggestion
M and N in a partial order, M is smaller than N in the Dershowitz-Manna ordering if M can be
```
I don't think it's worth mentioning the underlying type, but it's definitely worth mentioning that it's ordered